example/veneur/config.yaml

```diff
@@ -69,13 +69,10 @@ hostname: foobar
 # If true and hostname is "" or absent, don't add the host tag
 omit_empty_hostname: false
 
-# The address on which to listen for metrics sent to this instance over UDP,
-# leave empty to disable.
-udp_address: "localhost:8126"
-
-# The address on which to listen for metrics sent to this instance over TCP,
-# leave empty to disable.
-tcp_address: ""
+# The addresses on which to listen for metrics sent to this instance
```

Comment:
Similar to my comment below - since leaving these empty is equivalent to not defining them (which is not accidental), we should include `udp_address` and `tcp_address` in our examples, but set to the empty string. That allows us to consolidate `config.yaml` and `config_spec.yaml`.

Reply:
*That* I heavily disagree with. Where the `*config_spec.yaml` have to include backwards-compatible keys, I think the examples dir should be a shining beacon of excellent, modern and fresh-as-heck config that won't start hurting users when 2.0 rolls around (and we remove these keys).

Reply:
Having the spec be different from the yaml file defeats the entire point of the generation-as-build-validation that we're doing. There's nothing in the config file that will hurt users when 2.0 rolls around, because setting them to the zero value is a safe operation.